hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/java/org/apache/hadoop/fs/http/server/FSOperations.java

```diff
@@ -756,6 +760,10 @@ public JSONObject execute(FileSystem fs) throws IOException {
           return toJSON(
               StringUtils.toLowerCase(HttpFSFileSystem.DELETE_JSON), true);
         }
+        // Same is the behavior with Delete shell command.
+        // If moveToAppropriateTrash() returns false, file deletion
+        // is attempted rather than throwing Error.
+        LOG.error("Could not move {} to Trash, attempting removal", path);
```

Comment:
Let's lower this log level to `debug` instead.
`error` could generate a lot of noise if trash is not enabled here.

Reply:
Sure, sounds good. Let me do it right away.
Thanks